src/argilla/client/feedback/schemas/remote/metadata.py

```diff
@@ -28,13 +29,28 @@
 if TYPE_CHECKING:
     import httpx
 
-    from argilla.client.feedback.schemas.types import AllowedMetadataPropertyTypes
+    from argilla.client.feedback.schemas.types import AllowedMetadataPropertyTypes, AllowedRemoteMetadataPropertyTypes
     from argilla.client.sdk.v1.datasets.models import FeedbackMetadataPropertyModel
 
 
 class _RemoteMetadataProperty(RemoteSchema):
     @allowed_for_roles(roles=[UserRole.owner, UserRole.admin])
-    def delete(self) -> "AllowedMetadataPropertyTypes":
+    def _update(self: "AllowedRemoteMetadataPropertyTypes") -> None:
+        """Updates the `RemoteMetadataProperty` in Argilla.
+
+        Returns:
+            The updated `RemoteMetadataProperty` object.
+        """
+        updated_metadata_property = ArgillaMetadataPropertiesMixin.update(
+            client=self.client,
+            metadata_property_id=self.id,
+            title=self.title,
+            visible_for_annotators=self.visible_for_annotators,
+        )
+        self.__dict__.update(updated_metadata_property.__dict__)
+
+    @allowed_for_roles(roles=[UserRole.owner, UserRole.admin])
+    def _delete(self: "AllowedRemoteMetadataPropertyTypes") -> "AllowedMetadataPropertyTypes":
```

Comment:
Since `_update` is not being used. We can remove it. Also, we can remove the `_delete` and use directly the mixin helper for calling the API.